st/clitests/awss3api_spec.py

```diff
@@ -1913,6 +1913,28 @@ def get_md5_sum(fname):
 AwsTest('Aws can not enable versioning on non-existant bucket').put_bucket_versioning("seagate1", "Enabled")\
     .execute_test(negative_case=True).command_should_fail().command_error_should_have("NoSuchBucket")
 
+#******** GetObject with Bucket versioning enabled ********
+AwsTest('Aws can enable versioning on bucket').put_bucket_versioning("seagatebucket", "Enabled")\
+    .execute_test().command_is_successful()
+
+result = AwsTest('Aws can put object').put_object("seagatebucket", "1kfile", 1024)\
+    .execute_test().command_is_successful()
+
+versionId = result.status.stdout.split('\t')[1]
+
+result = AwsTest('Aws can get object with specific versionId').get_object("seagatebucket", "1kfile", version_id=versionId)\
+    .execute_test().command_is_successful().command_response_should_have(versionId)
+
+result = AwsTest('Aws can get object without specifing versionId').get_object("seagatebucket", "1kfile", version_id=versionId)\
```

Comment:
But we still specify the version_id here?

Reply:
Updated, thanks for pointing out.